app/helpers/tmp/icons_helper.rb

```diff
@@ -3,18 +3,6 @@ module Tmp::IconsHelper
   # rubocop:disable Rails/OutputSafety
   def reputation_icon(color:)
     '
-    <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can this whole file be deleted now? Or is it a placeholder for something?

Reply:
I failed to push my latest version where I had done this! Thanks.